lib/livekit/proto/livekit_agent_dispatch_twirp.rb

```diff
@@ -1,19 +1,23 @@
-# Code generated by protoc-gen-twirp_ruby 1.10.0, DO NOT EDIT.
-require 'twirp'
-require_relative 'livekit_agent_dispatch_pb.rb'
+# frozen_string_literal: true
+
+# Generated by the protoc-gen-twirp_ruby gem v1.2.0. DO NOT EDIT!
+# source: livekit_agent_dispatch.proto
+
+require "twirp"
+require_relative "livekit_agent_dispatch_pb"
 
 module LiveKit
   module Proto
-    class AgentDispatchServiceService < ::Twirp::Service
-      package 'livekit'
-      service 'AgentDispatchService'
-      rpc :CreateDispatch, CreateAgentDispatchRequest, AgentDispatch, :ruby_method => :create_dispatch
-      rpc :DeleteDispatch, DeleteAgentDispatchRequest, AgentDispatch, :ruby_method => :delete_dispatch
-      rpc :ListDispatch, ListAgentDispatchRequest, ListAgentDispatchResponse, :ruby_method => :list_dispatch
+    class AgentDispatchService < ::Twirp::Service
```

Comment:
did the generation change? or was this done manually?

Reply:
Looks to be generated, but it's been a while. I'll recheck.